package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sphere-node-sdk",
   "description": "Officially supported Node.js SDK library for working with the SPHERE.IO HTTP API, with OAuth2 support.",
-  "version": "1.24.0",
+  "version": "1.25.0",
```

Comment:
Not major version?

Reply:
Ah, sorry, I'm new to semver. Thought this would be breaking since anyone using old node versions need to update, @butenkor ? Would correct be `1.24.1` still?

Reply:
major would be 2.0.0

Reply:
Got it. Thanks a lot, @butenkor ! :)